tools/saveDistributionFunction.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import scipy.constants as const
+from scipy.io import savemat
+import sys, argparse, h5py
+sys.path.append('../py')
```

Comment:
I think we should change this to
```python
path = str((pathlib.Path(__file__).parent / '../py').resolve().absolute())
sys.path.append(path)
```
This will also require ``pathlib``:
```python
import pathlib
```